castle/validators/present.py

```diff
@@ -0,0 +1,10 @@
+from castle.exceptions import InvalidParametersError
+
+
+class ValidatorsPresent(object):
+
+    @staticmethod
+    def call(options, *args):
+        for key in args:
+            if options.get(key) is None or options.get(key) == '':
+                raise InvalidParametersError("{key} is missing or empty".format(key=key))
```

Comment:
do you need to test for empty array or empty object values?

Reply:
we need to validate options for non empty values specified by args

Reply:
what i meant is that empty array `[]` and empty dict `{}` are typically considered empty values like a blank string is `''`

Reply:
got it.. yes this is a very basic validation, just to handle the most obvious cases.. the api will return an error if wrong format will be provided like an array instead of a string.. but this is a great idea for a next enhancement  (I added that idea to sdk backlog )